tests/PHPUnit/Fixtures/UITestFixture.php

```diff
@@ -49,6 +53,10 @@ public function setUp()
             "idsite = 1"
         );
 
+        // for proper geolocation
+        LocationProvider::setCurrentProvider(LocationProvider\GeoIp\Php::ID);
+        IPAnonymizer::deactivate();
+
```

Comment:
Yeah, I don't see how it will slow the tests down, the GeoIP database will only be downloaded if it doesn't exist already. So it will happen once per test run (on travis). Is that what you're worried about?